lms/djangoapps/mobile_api/utils.py

```diff
@@ -5,13 +5,15 @@
 
 import functools
 
-from opaque_keys.edx.keys import CourseKey
-from courseware.courses import get_course_with_access
 from rest_framework import permissions
 from rest_framework.authentication import OAuth2Authentication, SessionAuthentication
 
+from opaque_keys.edx.keys import CourseKey
+from xmodule.modulestore.django import modulestore
+from courseware.courses import get_course_with_access
+
 
-def mobile_course_access(depth=0, verify_enrolled=True):
+def mobile_course_access(depth=0, verify_enrolled=True, bulk_operations=True):
```

Comment:
Is there an overhead to using `bulk_operations` if an endpoint makes only 1 call to the modulestore?